<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Menú Principal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QPushButton" name="b_usuarios">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuarios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_arduinos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Arduinos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_nodos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Nodos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_consulta">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Consulta</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="b_cerrarSesion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar Sesión</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_add">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_delete">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="user_list">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>501</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="user_change_pass">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_change_rank">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Permisos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cambios">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Cambios</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="arduino_list">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>501</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="arduino_turn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>87</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Encender</string>
    </property>
   </widget>
   <widget class="QListWidget" name="data_list">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_consultar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>87</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <zorder>b_consultar</zorder>
   <zorder>main_label</zorder>
   <zorder>groupBox</zorder>
   <zorder>b_cerrarSesion</zorder>
   <zorder>user_add</zorder>
   <zorder>user_delete</zorder>
   <zorder>user_list</zorder>
   <zorder>user_change_pass</zorder>
   <zorder>user_change_rank</zorder>
   <zorder>label_cambios</zorder>
   <zorder>arduino_list</zorder>
   <zorder>arduino_turn</zorder>
   <zorder>data_list</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
